configcatclient/user.py

```diff
@@ -24,10 +29,27 @@ def get_attribute(self, attribute):
         return self.__custom.get(attribute) if self.__custom else None
 
     def __str__(self):
-        dump = {
-            'Identifier': self.__identifier,
-            'Email': self.__data.get('Email'),
-            'Country': self.__data.get('Country'),
-            'Custom': self.__custom,
-        }
-        return json.dumps(dump, indent=4)
+        dump = OrderedDict([
+            ('Identifier', self.__identifier),
+            ('Email', self.__data.get('Email')),
+            ('Country', self.__data.get('Country'))
+        ])
+        if self.__custom:
+            dump.update(self.__custom)
+
+        filtered_dump = OrderedDict([(k, v) for k, v in dump.items() if v is not None])
+        return json.dumps(filtered_dump, separators=(',', ':'))
+
+    @staticmethod
+    def attribute_value_from_datetime(value):
+        if value is None or not isinstance(value, datetime):
+            return None
+
+        return str(utils.get_seconds_since_epoch(value))
+
+    @staticmethod
+    def attribute_value_from_list(value):
```

Comment:
I suggest the name `attribute_value_from_stringlist` as it would probably a more expressive name. Also, we may add other types of lists in the future.